app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,60 @@
+class AdminsController < ApplicationController
+  before_action :set_admin, only: [:edit, :update, :destroy]
+
+  # GET /admins
+  def index
+    @admins = Admin.all
+  end
+
+  # GET /admins/new
+  def new
+    @admin = Admin.new
+  end
+
+  # GET /admins/1/edit
+  def edit
+  end
+
+  # POST /admins
+  def create
+    @admin = Admin.new(admin_params)
+
+    respond_to do |format|
+      if @admin.save
+        format.html { redirect_to admins_path, notice: 'admin was successfully created.' }
```

Comment:
are we rendering the notice messages on the index page?

Reply:
sounds like a good idea